### PR TITLE
Cid 8589

### PR DESCRIPTION
**Pull request by Olie440**
Add Environment urls to the 11ty config

### CODE REVIEW

File: src/js/contact-page.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
   const MISSING_EMAIL_ERROR = 'Enter your email address';
   const INVALID_EMAIL_ERROR = 'Enter an email address in the correct format, like name@example.com';
   const MISSING_MESSAGE_ERROR = 'Enter your message';
-  const API_URL = 'https://api.dev.signin.nhs.uk/nhs-login-support/send-email';
```
Comment (61082EleanorBound) [inline src/js/contact-page.js:6]: Does the dev need to be dynamic to change for different dev environments?

File: src/_data/environment.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  EMAIL_API_URL: process.env.EMAIL_API_URL || 'https://api.dev.signin.nhs.uk/nhs-login-support/send-email',
```
Comment (61082EleanorBound) [inline src/_data/environment.js:2]: Same here about the dev environments being dynamic?

File: src/js/contact-page.js (hunk 1 above)
Comment (Olie440) [reply src/js/contact-page.js:6]: Yeah develop will point at dev whilst prod will point at live

File: src/_data/environment.js (hunk 2 above)
Comment (Olie440) [reply src/_data/environment.js:2]: See above

File: src/js/contact-page.js (hunk 1 above)
Comment (61082EleanorBound) [reply src/js/contact-page.js:6]: How will prod point at live?

**Pull request merged**